contracts/Trustcoin.sol

```diff
@@ -55,10 +55,10 @@ contract Trustcoin is OutgoingMigrationTokenInterface, ERC20TokenInterface, Safe
 
   // See ERC20
   function transferFrom(address _from, address _to, uint256 _value) external returns (bool) {
-    if (balances[_from] >= _value && allowed[_from][msg.sender] >= _value && _value > 0) {
+    if (balances[_from] >= _value && uint(allowed[_from][msg.sender]) >= _value && _value > 0) {
```

Comment:
can't this be a bug if `allowed[_from][msg.sender] < 0`? we'll convert to uint, and the result can be huge. I think we should first check `allowed[_from][msg.sender] > 0 && ...`